GroceryStore/Product.m

```diff
@@ -0,0 +1,42 @@
+//
+//  Product.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "Product.h"
+
+@interface Product ()
+
+@property (strong, nonatomic) NSString *productName;
+
+@property (strong, nonatomic) NSString *quantity;
+
+@end
+@implementation Product
+
+- (id)initWithProductName:(NSString *)productName andQuantity:(NSNumber *)quantity
+{
+    self = [super init];
+    if (self) {
+        self.productName = productName;
+        self.quantity = [NSString stringWithFormat:@"%@", quantity];
+    }
+    return self;
+}
+
++ (NSArray *)createInventoryFromResponseDict:(NSDictionary *)jsonDict {
+    
+    NSMutableArray *inventory = [NSMutableArray new];
```

Comment:
I don't know if it makes a difference, but I'd usually initialize a mutable array with this: `[NSMutableArray array];`